### ISSUE TITLE
Do you use WGAN?

### ISSUE THREAD

**Issue by jiujing23333** [duxingren14/DualGAN#4]
According to your paper, you use the loss format advocated by WGAN rather than the sigmoid cross-entropy loss used in the original GAN. 

But in this repo, it looks no WGAN. That's why?

**Comment by duxingren14** [comment]
Thanks for your question.

Actually the codes has evolved a few generations since the submission of
the ICCV paper. As more experiments imply that the original GAN seems to
generate better images (looks more natural), I replaced the WGAN loss here
with the original loss, though the training converges a bit slower. I would
probably do more experiments to confirm this and keep the codes being
improved. But so far the original GAN generates the best results. There are
other minor revisions that are different from the paper.

Thanks,

**Comment by nobodykid** [comment]
So the current version of code still use original GAN loss?

Is there a way to switch back to the original WGAN loss you used in the paper?
